slices/python3.yaml

```diff
@@ -0,0 +1,25 @@
+package: python3
+
+essential:
+  - python3_copyright
+
+slices:
+  core:
+    essential:
+      - python3.12_core
+
+  standard:
+    essential:
+      - python3.12_standard
+
+  utils:
+    essential:
+      - python3.12_utils
+    contents:
+      /usr/bin/pdb3:
```

Comment:
Do these files need to be symlinked to their python 3.12 alternatives in python3.12_utils?
ex. /usr/bin/pdb3 -> /usr/bin/pdb3.12

Reply:
They are already symlinks when installed from python3 package, and they are linked properly (pdb3 linked to pdb3.12, etc).